### ISSUE TITLE
is this a violation of our naming policy?

### ISSUE THREAD

**Issue by schristley** [airr-community/airr-standards#638]
I was looking at a GermlineSet and saw this. I thought our policy was lower snake case? Are we making exceptions for proper names (or stuff like ORCID)?

I kinda wonder why this isn't a CURIE, isn't it an external identifier?

```
          "acknowledgements": [
            {
              "acknowledgement_id": "",
              "name": null,
              "institution_name": null,
              "ORCID_id": null
            }
          ],
```

**Comment by javh** [comment]
Not that I know of (and wouldn't want to). I think it's just a typo.

**Comment by bussec** [comment]
I agree that it should be lower snake case, therefore #639 is fine with me. I also think that this should become a CURIE, as we now have support for ORCID in the CURIE section of the Schema. This can be addressed via #552 for v2.0.
